engine/src/main/java/org/terasology/engine/core/TerasologyEngine.java

```diff
@@ -253,7 +253,7 @@ public void initialize() {
         }
 
         double seconds = 0.001 * totalInitTime.elapsed(TimeUnit.MILLISECONDS);
-        logger.info("Initialization completed in {}sec.", String.format("%.2f", seconds));
+        logger.atInfo().addArgument(() -> String.format("%.2f", seconds)).log("Initialization completed in {}sec.");
```

Comment:
`String.format` complexity and impact are neglectable.
```suggestion
        logger.info("Initialization completed in {}sec.", String.format("%.2f", seconds)); //NOPMD
```